### ISSUE TITLE
Option 'hide' does not work

### ISSUE THREAD

**Issue by hxhieu** [lecar-red/bootstrapx-clickover#19]
Hi,

Sometime you may want to hide the Clickover after some action other than global click or timer and the default $.popover('hide') or $.clickover('hide') is buggy.
I have hacked my way little bit to add another option 'forcehide' and intercept it:

<pre><code>Line 188:
if (typeof option == 'string') {
    if (option === 'forcehide') data.clickery();
    else data[option]();
}</code></pre>

This is just a work around for me to get it done :)

Cheers,
Hugh

**Comment by lecar-red** [comment]
interesting, let me take a look.

Are you calling `hide` in your own event handler?

Something like:

``` javascript
$(function() { 
  $('body').on('click', '#otherbutton', function(e) { 
  $('#clickover').clickover('hide');
  e.preventDefault();
});
```

**Comment by hxhieu** [comment]
Yep, that what I do and the clickover does hide but next time when you trigger it, it won't show up. If you monitor on dev tools you will see the clickover div does get instantiated for a second then gets removed from the DOM tree straight

**Comment by lecar-red** [comment]
does that work differently if you do:

``` javascript
$('#clickover').data().clickover.hide()
```

**Comment by aturkelson** [comment]
I just ran in to this same issue - has there been any official update?

**Comment by kaxline** [comment]
I was able to get around this by artificially firing a 'click' event on an outside element.

``` javascript
$('#element-outside-clickover').trigger('click')
```

Total hack, but it works for now ...

**Comment by enginehouse** [comment]
$('#element-outside-clickover').clickover('hide').attr("data-clickover-open", 0);

should do it
